front/src/core/const.ts

```diff
@@ -9,12 +9,15 @@ export const SocketOuputMessageLiteral = {
 
 export const SocketEmitMessageTypes = {
   TRAINER_APPEND_TEXT: 'TRAINER_APPEND_TEXT', // Master creates a user story
+  TRAINER_SAVE_CONTENT: 'TRAINER_SAVE_CONTENT', // Master saves current content
```

Comment:
Not sure what this message is for

Reply:
Residue from previous iteration (before saving content on append). Removing.